install_kafka.yml

```diff
@@ -0,0 +1,92 @@
+---
+  - name: "Install Kafka"
+    hosts: kafkaserver1
+
+
+    roles:
+    - install-JDK-role
+
+    tasks:
+
+      #  - name: Install the latest version of JDK
+      #    become: yes
+      #    yum:
+      #     name: jdk-11
+      #     state: latest
+
+    
+       - name: Copy Kafka Binaries
+         become: yes 
+         become_user: dojokafka
+         copy:
```

Comment:
instead of `copy`, please use `template` 
and all relevant files should be in `jinja` format.  
variables should reside in vars (`defaults` or `vars` )

Reply:
Used templated residing in templated user respective roles